### DISCUSSION TITLE
Problem. With gifts for students

### DISCUSSION THREAD

**Discussion by wooslow** [community/community#140725] [GitHub Education]
### Hi everyone,

Hey everyone, guys, help me with a problem. I once activated Github Student Pack, but after some time I was deprived of benefits. Now I want to submit an application again, I go to the site, click on the button and I am taken to a page where I “create an organization”. What can I do? I'm trying to make a ticket in support, but it's just not created

**Comment by wooslow** [comment]
![image](https://github.com/user-attachments/assets/9e958f9e-8797-4bfc-92dd-bbcdf2695c88)

But there's also a sign here, but it's been up for 6 months now

**Comment by AimenDev** [comment]
If you're still having trouble, consider reaching out to GitHub via their Twitter support account: @GitHubHelp
